functions/exercise16/script.js

```diff
@@ -1 +1,11 @@
-"use strict";
\ No newline at end of file
+"use strict";
+
+function getUniqueCharactersFromString(string) {
```

Comment:
string is not a good name here - it is a type name, so it is better to rename it to something else.